app/models/report_filter.rb

```diff
@@ -5,38 +5,67 @@ class ReportFilter
     resolved
     account_id
     target_account_id
```

Comment:
In theory we could replace these with search_type and search_term (where the term is the `Account#acct`), but for now I think I'll leave these as is, since that requires changes elsewhere.

Reply:
I'll probably do up a follow up PR to change those inbound links to the reports list page to use the search capability instead, and deprecate `account_id` and `target_account_id` as parameters.